microsoft-edge/web-platform/release-notes/133.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Microsoft Edge 133 web platform release notes (Feb 2025)
+description: Microsoft Edge 133 web platform release notes (Feb 2025)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 01/31/2025
+---
+# Microsoft Edge 133 web platform release notes (Feb 2025)
+
+This article provides information about the new web platform features and updates in Microsoft Edge 133, which was released on February 6, 2025.
+
+To stay up to date and get the latest web platform features, download an Insiders preview version of Microsoft Edge. See [Microsoft Edge Insiders Channels](https://aka.ms/microsoftedge).
+
+<!-- ====================================================================== -->
+## Edge DevTools
+
+For Edge DevTools updates, see For Edge DevTools updates, see [What's New in Microsoft Edge DevTools](../../devtools-guide-chromium/whats-new/whats-new.md).
```

Comment:
```suggestion
For updates of Edge DevTools, see [What's New in Microsoft Edge DevTools](../../devtools-guide-chromium/whats-new/whats-new.md).
```
imo, don't need "Microsoft" b/c link text has it
@captainbrosset 
i'm likely to make these changes and push commits.